src/Module.php

```diff
@@ -133,6 +133,27 @@ class Module extends \yii\base\Module implements BootstrapInterface
      */
     public $traceLine = self::DEFAULT_IDE_TRACELINE;
 
+    /**
+     * @var string|array HTML title
+     * ```php
+     *  ...
+     *  'html_title'=> 'Custom Debug Title',
+     *  ...
+     * ```
+     * ```php
+     * 'html_title'=> [
+     *     'base-url'=> 'page title',
+     *     'http://qa.example.test'=> 'qa debugger'
+     * ]
+     * ```
+     */
+    public $html_title;
+
+    /**
+     * @var string
+     */
+    private $default_html_title = 'Yii Debugger';
```

Comment:
defaultPageTitle

Reply:
hardcoded return "Yii Debugger";
So no need for defaultPageTitle